neon_diana_utils/configuration.py

```diff
@@ -193,6 +195,38 @@ def make_keys_config(write_config: bool,
             config_confirmed = \
                 click.confirm("Is this configuration correct?")
 
+    vllm_config = dict()
+    if click.confirm("Configure VLLM?"):
+        config_confirmed = False
+        while not config_confirmed:
+            vllm_api_url = click.prompt("VLLM API URL", type=str)
+            vllm_connection_key = click.prompt("VLLM Connection Key", type=str)
+            vllm_hf_token = click.prompt("Hugging Face Auth Token", type=str)
+            vllm_role = click.prompt("VLLM Role",
+                                     type=str,
+                                     default="You are NeonLLM.")
+            vllm_context_depth = click.prompt("VLLM Context depth",
+                                              type=int,
+                                              default=4)
+            vllm_max_tokens = click.prompt("Maximum tokens in responses",
+                                           type=int,
+                                           default=512)
+            vllm_num_parallel_processes = click.prompt("Number of parallel processes",
+                                                       type=int,
+                                                       default=2)
```

Comment:
In the [huggingface demo](https://huggingface.co/spaces/neongeckocom/BrainForge-Demo-Test), we are moving towards exposing these as settings to adjust at runtime. Rather than using config, consider exposing these through the VLLM connector API

Reply:
I see this is consistent with other LLM endpoints. Refactoring this would be a larger change, requiring some consideration.